static/css/static.css

```diff
@@ -1560,7 +1560,7 @@ input#id_email{
     float: left;
 }
 
-#visualGardenPage .control-results-containe{
+#visualGardenPage .control-results-container{
```

Comment:
This is potentially a bug fix, or if the css failing to apply was no longer maintained, fixing it may produce bugs